trlx/data/configs.py

```diff
@@ -29,6 +29,8 @@ class ModelConfig:
     model_type : str # One of the architectures present in framework.model
     device : str = ''
     num_layers_unfrozen : int = -1
+    n_soft_tokens: int = None # soft prompt support
```

Comment:
You should be making a new model config for this so it isnt inherited by other model configs except your own.